### DISCUSSION TITLE
Booted method not working

### DISCUSSION THREAD

**Discussion by kieferjs** [livewire/livewire#4472] [1. Help]
I noticed the fix in 2.7.1 for the booted method with php artisan view:clear required to recompile the views.   

I still get an error that the booted method is unrecognized in my livewire (Method App\Http\Livewire\Leads\Index::booted does not exist.)

It is implemented in my livewire index file as follows:

    protected static function booted()
    {
        static::addGlobalScope(new AgentScope);
    }

currently using the following:
livewire/livewire v2.8.2
php 8.0.14

**Comment by kieferjs** [comment]
I believe I got around this issue by newing up a new model in the mount method instead of referencing another method "makenewtask" in the livewire file.  Because the model file has the agent scope, it still applied the scope with the static booted function removed in the livewire file.

**Comment by joshhanley** [comment]
This issue has been converted to a discussion as we are now using discussions to track bugs and ideas.
See this announcement for more details https://github.com/livewire/livewire/discussions/3147
